src/libs/actions/Policy.ts

```diff
@@ -200,6 +201,13 @@ Onyx.connect({
     callback: (val) => (allRecentlyUsedTags = val),
 });
 
+let allPolicyCategories: OnyxCollection<PolicyCategories> = {};
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.POLICY_CATEGORIES,
+    waitForCollectionCallback: true,
+    callback: (val) => (allPolicyCategories = val),
```

Comment:
```suggestion
    callback: (value) => (allPolicyCategories = value),
```

Reply:
this is fine, we seem to do it in a lot of other Onyx connections